Updated, modernized, and awesome

paytonrules opened the pull request.

paytonrules committed:
Updated, modernized, and awesome

rmulhol commented on libraries/shared/factories/storage/README.md:
```diff
@@ -1,89 +1,98 @@
 
-The current approach for caching smart contract storage diffs assumes that you are running a node that is writing contract storage diffs to a CSV file.
-The CSV file is expected to have 5 columns: contract address, block hash, block number, storage key, storage value.
+We have [a branch of go-ethereum (a.k.a geth)](https://github.com/makerdao/go-ethereum) that enables running a node that provides storage diffs via a websocket, emitting them as nodes are added to the chain, and is required for using `extractDiffs`. That command will continuously read from the websocket and persist the results to postgres, in the `public.storage_diffs` table to be transformed.
```
I think instead of "emitting them as nodes are added to the chain" we could say "as blocks are added" or "as transactions are applied on"

rmulhol commented on libraries/shared/factories/storage/README.md:
```diff
@@ -1,89 +1,98 @@
+	}
+	mappings, bidErr := loader.loadBidKeys(mappings)
+	if bidErr != nil {
+		return nil, fmt.Errorf("error adding bid keys to flap keys loader: %w", bidErr)
```
I don't know why these ones have the `%` highlighted in red 🤔 

Used to seeing something like that when something isn't closed (parens, quotes, etc), but I'm not seeing anything missing

paytonrules replied:
That's weird - but they look okay rendered. Something with github's diffing algorithm?

rmulhol commented on libraries/shared/factories/storage/README.md:
```diff
@@ -117,23 +126,50 @@ A contract-specific implementation of the repository interface enables the trans
-	Repository storage_diffs.Repository
+	Address    			common.Address
+	StorageKeysLookup 	KeysLookup
+	Repository 			Repository
```
Looks like the alignment is off here

paytonrules committed:
Update the Example to match

This is extremely out of date. I think I'd rather this use a real
example, but that was even more ornerous, so I've done my best to make
this fit. However I'm not sure it does.

paytonrules committed:
Fix link to event transformers docs
